File: backend/finder/src/main/java/ru/denis/finder/model/Match.java
package ru.denis.finder.model;

import jakarta.persistence.*;
import lombok.Data;

import java.time.LocalDateTime;

@Entity
@Data
public class Match {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne
    private UserProfile user1;
    @ManyToOne
    private UserProfile user2;

    private LocalDateTime matchedAt = LocalDateTime.now();
}

File: frontend/src/models/Match.ts
export interface Match {
  id: number;
  user1: {
    id: number;
    name: string;
    avatar?: string;
  };
  user2: {
    id: number;
    name: string;
    avatar?: string;
  };
  // Добавь другие поля, если нужно
} 